Day1Main.py

```diff
@@ -0,0 +1,25 @@
+from InputReader import InputReader
+from Day1.DayOnePartOne import FloorCounter
+from Day1.DayOnePartTwo import BasementFinder
+
+input_reader = InputReader("input1.txt")
+raw_input = input_reader.get_raw_input()
+
+floor_counter = FloorCounter(raw_input)
+parsed_input = floor_counter.parse_input()
+floor_counter.count_floors(parsed_input)
+print("The final floor at the end of the input:", floor_counter.get_final_floor())
+
+basement_index_finder = BasementFinder(raw_input)
+basement_index_finder.calculate_basement_entry(parsed_input)
+print("The step at which Santa enters the basement:", basement_index_finder.get_basement_entry())
+
+robs_challenged1 = FloorCounter("(((-2)+6((")
+robs_challenged1_input = robs_challenged1.parse_input()
+robs_challenged1.count_floors_with_skips(robs_challenged1_input)
+print("If Santa is lazy and uses an elevator, and is crazy and hates the number 8, the final floor is:", robs_challenged1.get_final_floor())
+
+robs_challenged2 = FloorCounter("(((((((((((((((((((((((((((((((((((((((((((((")
```

Comment:
nem tudom mi a kulonbseg a rob_challanged 1 es 2 kozott nyilvan ha elolvasom a feladat kiirast vagy a printed rajohetek de nem akarok azzal idot tolteni h mindent elolvassak a vilagon hatha rajovok h mi tortenik

talalj ket leiro nevet az elsore pl jo az elevator mert a print alapjan a business logicban is az egy elevator

es akkor most mondhatod h elevator az nem floorCounter ami jogos es jelezheti h vagy hianyzik egy class vagy rossz vminek a neve

ebben az esetben lehetne egy elevator classod ami kap egy floorCountert a ctorban
van egy get_final_final floor fve amibe beleteszed azokat a dolgokat amiket most a mainbol hivogatsz:

```
robs_challenged1 = FloorCounter("(((-2)+6((")robs_challenged1_input = robs_challenged1.parse_input()
robs_challenged1.count_floors_with_skips(robs_challenged1_input)
print("If Santa is lazy and uses an elevator, and is crazy and hates the number 8, the final floor is:", robs_challenged1.get_final_floor())
```

nyilvan ne konkretan ezt de majd az uj verzio oda vonatkozo reszet

masik lehet superstitious elevator ami megkapja h miket ugorjon es a skip logic az elevatorban van nem a floor counterben mert a floor counter nem skippel hanem floor countol

nem baj ha egy classod nem buzi nagy es nem old meg mindent

egy feladatkort oldjon meg, ott oldjon meg mindent, de ne tobbet